docker/run_tests_in_docker.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Script that sets up the docker environment to run the tests in and runs the
+# tests.
+
+# Pass --shell to run a shell in the test container.
+
+# Failures should cause setup to fail.
+set -v -e -x
+
+DC="$(which docker-compose)"
+
+# Use the same image we use for building docker images because it'll be cached
+# already.
+BASEIMAGENAME="local/pontoon_base"
+
+# Start services in background (this is idempotent).
+echo "Starting services in the background..."
+${DC} up -d postgresql
+
+# If we're running a shell, then we start up a test container with . mounted
+# to /app.
+if [ "$1" == "--shell" ]; then
+    echo "Running shell..."
+
+    docker run \
+           --rm \
+           --volume "$(pwd)":/app \
+           --workdir /app \
+           --network pontoon_default \
+           --link pontoon_postgresql_1 \
+           --env-file ./docker/config/webapp.env \
+           -e LOCAL_USER_ID=$UID \
+           --tty \
+           --interactive \
+           local/pontoon /bin/bash
+
+else
+    # Create a data container to hold the repo directory contents and copy the
+    # contents into it.
+    if [ "$(docker container ls --all | grep pontoon-tests)" != "" ]; then
+        echo "Removing previously existing container"
+        docker rm pontoon-tests
```

Comment:
This is the easiest way to ensure that the tests run without old content. For example, in cases when we remove files from the repo, if we don't make sure we delete the entire content and re-copy it, then the tests will still run the removed files. That hit us on socorro, and remove the container is the fastest and simplest way to circumvent this.